Assets/LeapMotion/Scripts/VR/LeapVRTemporalWarping.cs

```diff
@@ -182,17 +182,28 @@ protected struct TransformData {
     }
   
     protected void Start() {
-      //Get a callback right as rendering begins for this frame so we can update the history and warping.
-      LeapVRCameraControl.OnValidCameraParams += onValidCameraParams;
-  
-      deviceInfo = provider.GetDeviceInfo();
+
+      Controller controller = provider.GetLeapController();
+      controller.Device += OnDevice;
+    }
+    
+    protected void OnDevice(object sender, DeviceEventArgs args) {
+       deviceInfo = provider.GetDeviceInfo();
       if (deviceInfo.type == LeapDeviceType.Invalid) {
         Debug.LogWarning("Invalid Leap Device -> enabled = false");
         enabled = false;
         return;
       }
+      //Get a callback right as rendering begins for this frame so we can update the history and warping.
+      LeapVRCameraControl.OnValidCameraParams += onValidCameraParams;
```

Comment:
Subscribing here means that every time the user unplugs and replugs their device in while running the app, there will be an extra subscription added to OnValidCameraParams